### PR TITLE
Add semistandard rules

### PR DESCRIPTION
**Pull request by elenatorro**
This is a proposal to add `semistandard` linter to this repo in order to have similar rules in all our frontend projects. I have left the rules we had in this project (as indent spaces), and we can discuss which ones we want to add / remove.

### CODE REVIEW

**Comment by Jesus89** [comment]
I'm OK running lint --fix with the current rules.

**Comment by davidmanzanares** [comment]
:+1: But I would override the indentation to `4` spaces. Every line is going to change without that.

**Comment by Jesus89** [comment]
OK for me. And let's install a linter plugin for eslint in our IDEs (to avoid linter failing in CI)

**Comment by Jesus89** [comment]
UPDATE:
- Tests fixed
- Ignore only `no-tabs` and `import/no-duplicates` rules
- Rename SQL, MVT and GeoJSON since they are acronyms

**Comment by elenatorro** [comment]
Before merging, since there are A LOT of changes, I think it is better to have more eyes on this CR 👀 cc @jgoizueta @IagoLast

**Comment by IagoLast** [comment]
Why leaving the `eslint:recommended` rules ? Can we just extend semistandard?

File: .eslintrc.json [hunk 1]
```diff
@@ -30,6 +34,8 @@
         ],
         "no-console": "off",
         "no-useless-escape": "off",
-        "no-var": "error"
+        "no-var": "error",
+        "no-new": "off",
+        "import/no-duplicates": "off"
```
Comment (davidmanzanares) [inline .eslintrc.json]: Why this?

Comment (Jesus89) [reply .eslintrc.json]: This is not required anymore since we have separated SVG files with proper spacing.

**Comment by elenatorro** [comment]
🚀

File: test/common/util.js [hunk 2]
```diff
@@ -39,7 +39,11 @@ function getName (file) {
     );
 }
 
+<<<<<<< HEAD
```
Comment (IagoLast) [inline test/common/util.js]: !!!

File: test/common/util.js [hunk 3]
```diff
@@ -51,7 +55,11 @@ function takeReference (file, template, asyncLoad) {
     }
 }
 
+<<<<<<< HEAD
```
Comment (IagoLast) [inline test/common/util.js]: !!!

File: test/common/util.js (hunk 2 above)
Comment (elenatorro) [reply test/common/util.js]: There was a problem with `code` but it was fixed in a commit before the merge.

**Pull request merged**